lib/dor/was_crawl/cdxj_generator_service.rb

```diff
@@ -33,7 +33,8 @@ def cdx_file_name(warc_file_name)
       def generate_cdx_for_one_warc(warc_file_name)
         cdx_file_path  = cdx_file_name(warc_file_name)
         warc_file_path = "#{druid_base_directory}/#{warc_file_name}"
-        cmd_string = "#{Settings.cdxj_indexer.bin} #{warc_file_path} --output #{cdx_file_path} --dir-root #{Settings.was_crawl_dissemination.stacks_collections_path} --post-append 2>> log/cdx_indexer.log"
+        cmd_string = "export TMPDIR=#{Settings.cdxj_indexer.tmpdir} " \
```

Comment:
Can you just do following?
```
cmd_string = "TMPDIR=#{Settings.cdxj_indexer.tmpdir} #{Settings.cdxj_indexer.bin} #{warc_file_path} --output #{cdx_file_path} --dir-root #{Settings.was_crawl_dissemination.stacks_collections_path} --post-append 2>> log/cdx_indexer.log"
```

Reply:
Thanks, right, that would be simpler. Updated.

Reply:
@justinlittman would you look at the update when you have a chance and approve or comment?